roles/ipaclient/library/ipaclient_get_otp.py

```diff
@@ -135,8 +135,7 @@
 
 def get_host_diff(ipa_host, module_host):
     """
-    Compares two dictionaries containing host attributes and builds a dict
-    of differences.
+    Compare two host dicts and builds a dict of differences.
```

Comment:
I think using "Build a dictionary with the differences from two host dictionaries." still shows what is being done, and doesn't look like the function is doing two different things.